cylc/flow/workflow_files.py

```diff
@@ -320,10 +320,9 @@ class RemoteCleanQueueTuple(NamedTuple):
 """
 
 SUITERC_DEPR_MSG = (
-    f"Back-compat mode turned ON for Cylc 7 '{WorkflowFiles.SUITE_RC}' "
-    "files.\n"
-    f"Do NOT rename to '{WorkflowFiles.FLOW_FILE}' without "
-    "upgrading to Cylc 8 syntax."
+    f"Deprecated Cylc 7 '{WorkflowFiles.SUITE_RC}' filename detected."
+    f" Upgrade\nto Cylc 8 syntax BEFORE renaming the file "
```

Comment:
```suggestion
    f" Upgrade to Cylc 8 syntax BEFORE renaming the file "
```

Reply:
Actually, I can't get nice line breaking in the terminal without newlines in the string. And now I've added more to the message ("Backward compatibility is limited - see Cylc 8 documentation"). So I've gone with newlines between the three sentences.